channels/kr.m3u

```diff
@@ -32,25 +32,19 @@ http://1.222.207.80:1935/live/cjbtv/playlist.m3u8
 #EXTINF:-1 tvg-id="CTSgidoggyoTV.kr" tvg-country="KR" tvg-language="Korean" tvg-logo="http://www.bbsi.co.kr/HOME/img/common/imgLogo.png" group-title="",CTS기독교TV (720p)
 https://d34t5yjz1ooymj.cloudfront.net/out/v1/875039d5eba0478fa8375a06b3aa5a37/index.m3u8
 #EXTINF:-1 tvg-id="EBS1.kr" tvg-country="KR" tvg-language="Korean" tvg-logo="https://global.ebs.co.kr/global/img/sub/Channels_ebs1.png" group-title="",EBS 1 (400p)
-http://ebsonair.ebs.co.kr/ebs1familypc/familypc1m/playlist.m3u8
+https://ebsonair.ebs.co.kr/ebs1familypc/familypc1m/playlist.m3u8
```

Comment:
it has http for a reason. No longer plays in VLC if you put it this way. And goes for infinite load the other way. Needs verifying

Reply:
oddly enough those work with their player, moving them to http.